bin/oref0-setup.sh

```diff
@@ -831,6 +805,11 @@ if prompt_yn "" N; then
     echo Running apt-get autoclean
     sudo apt-get autoclean
 
+    # create pump.ini
+    echo '[device "pump"]' > pump.ini
+    echo "serial = $serial" >> pump.ini
+    echo "radio_locale = $radio_locale" >> pump.ini
```

Comment:
Would it be better to move just this one line after the else block?

Reply:
it's required for `nightscout autoconfigure-device-crud ` to have a good pump.ini

Reply:
That doesn’t mean we should create it manually and then also create it with openaps device add pump.

Reply:
i just moved your lines from https://github.com/openaps/oref0/blame/dev/bin/oref0-setup.sh#L1071 a bit more up (they were in an `else ` part of Medtronic CGM). radio_locale is generated several times and causes invalid pump.ini with a fresh 0.7.0-dev install. You created these lines with https://github.com/openaps/oref0/commit/13068f32ef3829967d2c4a0cf82e68131cf0ec71

Reply:
The first two of those lines are normally created by the openaps device add pump commands. If you’re not replacing the openaps commands, you only need to move the third line outside the else. I’m not sure if adding all three adds the first two twice, or if openaps device add ignores them if they already exist.